src/compiler/commandLineParser.ts

```diff
@@ -1088,11 +1088,11 @@ namespace ts {
     }
 
     /** Tuple with error messages for 'unknown compiler option', 'option requires type' */
-    type ParseCommandLineWorkerDiagnostics = [DiagnosticMessage, DiagnosticMessage];
+    type ParseCommandLineWorkerDiagnostics = [DiagnosticMessage, DiagnosticMessage, DiagnosticMessage];

```

Comment:
At this point it might be time for actual names https://twitter.com/SeaRyanC/status/1052986385974480896

Reply:
👍  done